pkgs/by-name/vz/vzic/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  glib,
+  tzdata,
+  libical,
+}:
+stdenv.mkDerivation rec {
+  pname = "vzic";
+  version = "2024a";
+
+  src = fetchFromGitHub {
+    owner = "libical";
+    repo = "vzic";
+    rev = "4a87e2bed237585cbcf209507d0b82c023113633";
+    hash = "sha256-zRy62SJa8XUBPnoW/fgANKZaatNWqmXka4LgRUlbLzw=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    tzdata
+    glib
+    libical
+  ];
+  env = {
+    OLSON_DIR = "tzdata" + version;
+    PRODUCT_ID = "-//NixOS//NONSGML Citadel calendar//EN";
+    TZID_PREFIX = "/NixOS/Olson_%D_1/";
+  };
+
+  # Install vzic manually because there's no install rule in Makefile
+  installPhase = ''
+    runHook  preInstall
+    mkdir -p $out/share/vtimezone
+    ./vzic --output-dir $out/share/vtimezone
+
+    install -Dm744 vzic $out/bin/vzic
+    runHook  postInstall
+  '';
+
+  meta = {
+    homepage = "https://github.com/libical/vzic";
+    description = "A program to convert the IANA timezone database files into VTIMEZONE files compatible with the iCalendar specification";
+    mainProgram = "vzic";
+    license = with lib.licenses; [ gpl2Plus ];
+    maintainers = with lib.maintainers; [ moraxyc ];
+    platforms = lib.platforms.unix;
+  };
```

If we need to execute what we built, then we'll need to have an emulator available. If there is none, then the derivation cannot be built.

```suggestion
    broken = !stdenv.hostPlatform.emulatorAvailable buildPackages;
  };
```